### PR TITLE
Author tests to ensure XML messages comply with schema

### PR DESCRIPTION
**Pull request by timlegge**
Depends on #160 simply adds an author test to ensure that the XML matches the schema

### CODE REVIEW

File: t/author/01-confirm-schema.t [hunk 1]
```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env perl
```
Comment (waterkip) [inline t/author/01-confirm-schema.t:1]: If you put these in an `xt` directory instead of `t` we don't need to set the AUTHOR_TESTING env var.

**Review by waterkip — APPROVED** [review]

File: t/author/01-confirm-schema.t (hunk 1 above)
Comment (timlegge) [reply t/author/01-confirm-schema.t:1]: I will try it.

you mean xt/t ?

Comment (waterkip) [reply t/author/01-confirm-schema.t:1]: no, just xt

File: t/author/01-confirm-schema.t [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+BEGIN {
+  unless ($ENV{AUTHOR_TESTING}) {
```
Comment (waterkip) [inline t/author/01-confirm-schema.t:4]: You can also do something like this:
if (!$ENV{AUTHOR_TESTING}) {
    plan skip_all => 'Author test skipped';
}

Comment (timlegge) [reply t/author/01-confirm-schema.t]: @waterkip resolved I think

**Pull request merged**